docs/animations/Blur.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Blur animation behavior
 author: nmetulev
-ms.date: 08/20/2017
 description: The UWP Community Toolkit Blur animation behavior selectively blurs a XAML element by increasing or decreasing pixel size
 keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, blur, blur animation
 ---
 
 # Blur
 
-The Blur animation blurs a XAML element by increasing or decreasing pixel size. Blur animation is applied to all the XAML elements in its parent control/panel. Blur animation doesn't affect the functionality of the control.
+The [Blur animation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions) blurs a XAML element by increasing or decreasing pixel size. Blur animation is applied to all the XAML elements in its parent control/panel. Blur animation doesn't affect the functionality of the control.
```

Comment:
Possible wrong link? Or do you want to link to https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions#Microsoft_Toolkit_Uwp_UI_Animations_AnimationExtensions_BlurEffect

Reply:
What if we link to Blur method? https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.animations.animationextensions#Microsoft_Toolkit_Uwp_UI_Animations_AnimationExtensions_Blur_Microsoft_Toolkit_Uwp_UI_Animations_AnimationSet_System_Double_System_Double_System_Double_